addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar_tests.js

```diff
@@ -180,7 +180,9 @@ QUnit.test('attachment counter while loading attachments', async function (asser
         threadId: 100,
         threadModel: 'res.partner',
     });
-    await this.createChatterTopbarComponent(chatter);
+    await afterNextRender(async () => {
+        await this.createChatterTopbarComponent(chatter);
+    });
```

Comment:
I think in this case it would make sense to have `afterNextRender` standalone since the render is resulting from the timeout and not the from the create itself.

And with that, it should be possible to check the loading mechanism in the test without any extra lost time:
- after create, ensure the parent is rendered
- and ensure there is no loader yet and no data

and afterNextRender:
- ensure there is the loader